src/solver/libModelObject/include/antares/solver/libObjectModel/constraint.h

```diff
@@ -32,11 +32,24 @@ namespace Antares::Solver::ObjectModel
 class Constraint
 {
 public:
-    Constraint();
-    ~Constraint() = default;
+    Constraint(std::string name, Expression expression):
+        id_(std::move(name)),
+        expression_(std::move(expression))
+    {
+    }
+
+    const std::string& Id() const
+    {
+        return id_;
+    }
+
+    Expression expression() const
```

Comment:
mismatch between the getter name style

Reply:
Yes. It can't be called Expression() because it clash with the type _Expression_